bin/ghe-backup

```diff
@@ -237,7 +237,7 @@ if [ "$GHE_BACKUP_STRATEGY" = "rsync" ]; then
 fi
 
 if [ "$GHE_PARALLEL_ENABLED" = "yes" ]; then
-  $GHE_PARALLEL_COMMAND $GHE_PARALLEL_COMMAND_OPTIONS -- "${commands[@]}"
+  "$GHE_PARALLEL_COMMAND" "${GHE_PARALLEL_COMMAND_OPTIONS[@]}" -- "${commands[@]}"
```

Comment:
The use of in-quote expanded array elements is _much_ safer that a non-quoted string.